src/main/imports/moviefun-route.json

```diff
@@ -0,0 +1,30 @@
+//api/route
```

Comment:
This file throw 400 error, I instead generated a new one that worked fine for the purpose:

```
//api/route
{
    "headers": [],
    "profileRequiresAll": true,
    "roles": [],
    "profiles": [
        {
            "displayName": "Movie fun OAuth2",
            "name": "Movie fun OAuth2",
            "description": "This OAuth2 profile is used to issue JWT tokens for the movie fun application.",
            "attributes": {},
            "id": "movie-fun-oauth2"
        }
    ],
    "description": "This route is used to proxy the Movie fun API through the Gateway. Additionally, it will enforce the JWT validation using the specified OAuth2 profile.",
    "index": 5,
    "modRewrite": "# this is to secure the REST API and needs to be first\n\nRewriteRule \"^/?moviefun/rest/(.*)$\" \"http://host.docker.internal:8020/moviefun/rest/$1\" [QSA,P,NE,auth]\n\n#this one is more permissive. We could filter only static resources or exclude the /rest but as it comes on second, that's fine\nRewriteRule \"^/?moviefun/(.*)$\" \"http://host.docker.internal:8020/moviefun/$1\" [QSA,P,NE]",
    "lastMatchPattern": "^/?moviefun/(.*)$",
    "roleRequiresAll": true,
    "tags": [
        {
            "name": "api",
            "attributes": {},
            "id": "api"
        },
        {
            "name": "fun",
            "attributes": {},
            "id": "fun"
        },
        {
            "name": "jwt",
            "attributes": {},
            "id": "jwt"
        },
        {
            "name": "movie",
            "attributes": {},
            "id": "movie"
        }
    ],
    "name": "Movie fun API",
    "id": "movie-fun-api"
}
```